magma/backend/coreir_.py

```diff
@@ -91,8 +91,12 @@ def wrap_if_named_type(self, port, definition):
             wrapped = self.wrap_if_named_type(port[0], definition)
             if not wrapped:
                 return False
-            for t in port[1:]:
-                self.wrap_if_named_type(t, definition)
+            # TODO: Magma doesn't support length zero array, so slicing a
```

Comment:
it might be cleaner to just change this to `for t in port.ts[1:]` rather than the explicit check here. Since we never really need a handle on the array returned by `port[1:]`, just its underlying `ts` this might be ok. Not perfect either but seems slightly better than the len check. If you do stick with the len check, maybe prefer

```python
if len(port) == 1:
    return True
for t in port[1:]:
    ...
return True
```